.probo.yaml

```diff
@@ -53,6 +53,7 @@ steps:
       - $SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush user:role:add az_content_editor azuseradmin
       - $SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush user:role:add az_content_admin azuseradmin
       - $SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush user:role:add az_user_admin azuseradmin
+      - $SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush -y -n config:set az_cas.settings disable_login_form false
```

Comment:
If the setting stays disabled by default (as it is now), this step won't actually be needed.

Reply:
Disabled by default makes sense to me.